src/Joonasw.AspNetCore.SecurityHeaders/Csp/Builder/CspBuilder.cs

```diff
@@ -82,9 +82,25 @@ public void ReportViolationsTo(string uri)
             if(uri.Length == 0) throw new ArgumentException("Uri can't be empty", nameof(uri));
 
             _options.ReportUri = uri;
-        }
+		}
+
+		/// <summary>
+		/// Configures the middelware to include the X-Header.
```

Comment:
The summary text could be: "Includes the X-Content-Security-Policy header in responses." You could also write about the browser support it offers.